_includes/tutorials/anomaly-detection/ksql/markup/dev/create-accounts-to-monitor-table.adoc

```diff
@@ -0,0 +1,21 @@
+For this use case, let's say that a single transaction to one of the companies in the suspicious_names table is probably okay, but multiple transactions to one or more of those companies in a 24-hour period is an anomaly. ksqlDB gives us the ability to see if any anomalies are present for a particular user with the following query.
+
+[source,sql]
+----
+CREATE TABLE accounts_to_monitor
+    WITH (kafka_topic='accounts_to_monitor', partitions=1, value_format='JSON') AS
+    SELECT TIMESTAMPTOSTRING(WINDOWSTART, 'yyyy-MM-dd HH:mm:ss Z') AS WINDOW_START, // <1>
+           TIMESTAMPTOSTRING(WINDOWEND, 'yyyy-MM-dd HH:mm:ss Z') AS WINDOW_END,
+           USERNAME
+    FROM suspicious_transactions
+    WINDOW TUMBLING (SIZE 24 HOURS) // <2>
+    GROUP BY USERNAME
+    HAVING COUNT(*) > 3; // <3>
+----
++++++
+<div class="colist arabic">
+<p><i class="conum" data-value="1"></i><b>1</b> The fields `WINDOW_START` and `WINDOW_END` tell us what interval of time suspicious activity occurred.</p>
+<p><i class="conum" data-value="2"></i><b>2</b> The <code>WINDOW TUMBLING</code> part of the query allows us to do an aggregation with distinct time boundaries. In this case our window is fixed at a length of 24 hours, does not allow gaps, and does not allow overlapping. Other types of windows are explained in the "Collect data over time" section of Kafka-Tutorials. For more in-depth descriptions and visualizations, checkout the <a href="https://docs.ksqldb.io/en/latest/concepts/time-and-windows-in-ksqldb-queries/#windows-in-sql-queries"><strong>ksqlDB documentation about windows</strong></a>. </p>
+<p><i class="conum" data-value="3"></i><b>3</b> The last two lines of the query address how you would determine if a user had multiple suspicious transactions. This aspect of the query says, in essence, if any user has greater than 3 suspicious transactions during the window, emit an event to the accounts_to_monitor table. </p>
+</div>
```

Comment:
Super minor nit: It wasn't obvious to me at first to create this table. Maybe add a sentence at the end, telling the user explicitly to create the table.  But only if you want to. Again this is a minor subjective comment.

Reply:
That's a great point, I did add a clause there.